x-pack/plugins/infra/index.ts

```diff
@@ -18,7 +18,7 @@ export function infra(kibana: any) {
     id: APP_ID,
     configPrefix: 'xpack.infra',
     publicDir: resolve(__dirname, 'public'),
-    require: ['kibana', 'elasticsearch'],
+    require: ['kibana', 'elasticsearch', 'metrics'],
```

Comment:
Please note that I don't really know if this is necessary or not. It works without this change but that might be luck.